content/en/docs/aboutjetstack/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "About Jetstack"
+linkTitle: "About Jetstack"
+weight: 90
+type: "docs"
+---
+
+cert-manager was originally created by [Jetstack](https://jetstack.io), and
+a lot of the development is still sponsored by us.
```

Comment:
the `by us` sounds weird as it suddenly switches to the first person as jetstack. Could we reword this?

Reply:
yeah it is a bit weird, but the rest of the page is in first person:

> We offer:

I will try and rewrite it

Reply:
Updated, PTAL